dev-tools/command-line-advanced.md

```diff
@@ -208,60 +208,47 @@ This lesson helps you create an executable script. It will read information from
 2. Create a small file with the filename 'lunch' (not lunch.txt) by typing `touch lunch`.
 
 3. Using a text editor of your choice, modify the contents of this brand-new `lunch` file so that it contains the following text:
-
-   ```bash
-   #!/bin/bash
-
-   lunch=$1
-   echo $lunch is for lunch
-   ```
-
-   `$1` refers to the first argument a user will pass into the lunch program.
-   Save the file and, if you're using a command line text editor, exit it.
+<<<<<<< HEAD:command-line/command-line-advanced.md
```

Comment:
This is a merge conflict marker and is not meant to be included in the actual committed file. Please remove and make sure you haven't included duplicate text or accidentally removed a section. It's hard to tell from the diff.